src/main/webapp/assets/modules/contest-detail/registrants/challengeRegistrantsTab.js

```diff
@@ -0,0 +1,37 @@
+techlooper.directive('funnelManagement', function () {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/contest-detail/registrants/funnelManagement.html",
+      link: function (scope, element, attr, ctrl) {
+      }
+    };
+  })
+  .directive('registrationList', function () {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/contest-detail/registrants/registrationList.html",
+      link: function (scope, element, attr, ctrl) {
+      }
+    };
+  })
+  .directive('qualifyAllToNextPhase', function () {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/contest-detail/registrants/qualifyAllToNextPhase.html",
+      link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.